NomadUI/Widgets/NUICoreWidgets.cpp

```
@@ -293,20 +295,80 @@ NUITabBar::NUITabBar() = default;
+            renderer.fillRoundedRect(tabRect, radius - inset, hover);
+        }
+
+        const auto textColor = isActive ? textActive : textInactive;
+        const auto textSize = renderer.measureText(tab.label, fontSize);
+        const float textX = std::round(tabRect.x + (tabRect.width - textSize.width) * 0.5f);
+        const float textY = std::round(tabRect.y + (tabRect.height - textSize.height) * 0.5f);
+        renderer.drawText(tab.label, NUIPoint(textX, textY), fontSize, textColor);
+    }
 }
 
 bool NUITabBar::onMouseEvent(const NUIMouseEvent& event)
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)